### PR TITLE
v2.1.3 (1)

### PR DESCRIPTION
**Pull request by losh11**

### CODE REVIEW

File: breadwallet/BRAPIClient+Wallet.swift [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-private let ratesURL = "https://litecoin.com/api/v1/rates"
+private let ratesURL = "https://api.loafwallet.com/api/rates"
```
Comment (thrasher-) [inline breadwallet/BRAPIClient+Wallet.swift]: This doesn't appear to be working for me. Was the DNS record just updated?

File: breadwallet/src/Platform/BRWebViewController.swift [hunk 2]
```diff
@@ -67,10 +66,22 @@ import WebKit
...
+                        }
+                    }
+                    return Date()
+                }
+                let walletAddress = walletManager.wallet?.receiveAddress
+                let currencyCode = Locale.current.currencyCode
+                let uuid = UIDevice.current.identifierForVendor!.uuidString
+
+                let address = getAddress(appInstallDate: appInstallDate, walletAddress: walletAddress, currencyCode: currencyCode, uuid: uuid)
+                print(address)
```
Comment (thrasher-) [inline breadwallet/src/Platform/BRWebViewController.swift]: I guess this was just used for debugging and can be removed.

File: breadwallet/src/Platform/BRWebViewController.swift [hunk 3]
```diff
@@ -80,29 +91,32 @@ import WebKit
...
     
+    private func getAddress(appInstallDate: Date?, walletAddress: String?, currencyCode: String?, uuid: String?) -> String {
+        guard let appInstallDate = appInstallDate else { return "" }
+        guard let walletAddress = walletAddress else { return "" }
+        guard let currencyCode = currencyCode else { return "" }
+        guard let uuid = uuid else { return "" }
+        
+        let timestamp = Int(appInstallDate.timeIntervalSince1970)
+        
+        return "https://buy.loafwallet.org/?address=\(walletAddress)&code=\(currencyCode)&idate=\(timestamp)&uid=\(uuid)"
```
Comment (thrasher-) [inline breadwallet/src/Platform/BRWebViewController.swift]: This DNS record doesn't appear to be working for me too.

**Comment by kcw-grunt** [comment]
All of the fixes are included in v2.2.0 which is merged into master...mind if I delete the branch?

**Comment by losh11** [comment]
Okay to delete unnecessary branches.

**Pull request closed without merging**